### PR TITLE
Michael, TK, Ahmed, Adrien - Implement Hand Unit Tests

### PR DESCRIPTION
**Pull request by Mchan2003**

### CODE REVIEW

File: subsystem-arm/test/hand/hand-modes_test.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            CHECK_EQ(args.pinky_angle, original_commands.pinky_angle);
+            CHECK_EQ(args.ring_angle, original_commands.ring_angle);
+            CHECK_EQ(args.middle_angle, original_commands.middle_angle);
+            CHECK_EQ(args.index_angle, original_commands.index_angle);
+            CHECK_EQ(args.thumb_angle, original_commands.thumb_angle);
+        }
+
+        SECTION("2.1 Should set angles to minimum angle when in close mode")
+        {
+            hand_arguments original_commands{30, 30, 30, 30, 30};
```
Comment (naterpotatoers) [inline subsystem-arm/test/hand/hand-modes_test.cpp:25]:
```suggestion
            hand_arguments original_commands{30, 30, 30, 30, 30};
```
Swap out the random special numbers with the static constexpr variables

Comment (naterpotatoers) [reply subsystem-arm/test/hand/hand-modes_test.cpp:25]: do the same with the other special numbers

**Review by naterpotatoers — APPROVED** [review]

**Review by naterpotatoers — APPROVED** [review]

**Comment by naterpotatoers** [comment]
need to fix broken test

**Comment by kammce** [comment]
Why is the title so long? Why so many names?

**Comment by naterpotatoers** [comment]
Stale branch closing for now

**Pull request closed without merging**